docs/developer-docs/latest/update-migration-guides/update-version.md

```diff
@@ -4,121 +4,64 @@ description: The following documentation covers how to upgrade your application
 canonicalUrl: https://docs.strapi.io/developer-docs/latest/update-migration-guides/update-version.html
 ---
 
-# Update Strapi version
+## Upgrade Strapi version
 
-In this guide you will learn how to upgrade your application to the latest version of Strapi.
-
-:::note
-When a new version of Strapi is available, you will be notified both in your terminal, and in the admin panel. <br>
-Also note that you can check the **Settings > Global settings - Application** section of the admin panel to have more information on:
-
-- The versions of Strapi and Node you are currently using.
-- If relevant, the versions we recommend you to upgrade to.
-
-:::
+Strapi periodically releases code improvements through upgrades. Upgrades contain no breaking changes and are announced in both the terminal and in the administration panel.  
```

Comment:
Perhaps we could mention migrations here? The documentation says "Upgrades contain no breaking changes", I think it would make sense to compare to migrations which do contain breaking changes and then refer to migration guides. What do you think?

Reply:
Yes! I went back and forth on this idea, but I am happy to add it to the revision.